docs/redirects.asciidoc

```diff
@@ -1,14 +0,0 @@
-["appendix",role="exclude",id="redirects"]
```

Comment:
It would be critical to keeping this page as it supports the redirect strategy of the book. Keeping this file in the repo makes it possible for us to remove/rename pages and page IDs without breaking the links that point to those pages.

Would it be possible to add it back? If it would, then please add the following content to it (under Deleted pages):

```
[role="exclude",id="experimental_and_beta_apis"]
== Experimental and beta APIs

This page has been removed.
```

Reply:
Done